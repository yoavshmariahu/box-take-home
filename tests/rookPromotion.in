k a2
K e4
r b3
R d3

[G S B P]
[p b s g]

move b3 b5 promote
move d3 d1 promote
