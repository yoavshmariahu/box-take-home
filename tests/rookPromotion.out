UPPER player action: move d3 d1 promote
5 |__|+r|__|__|__|
4 |__|__|__|__| K|
3 |__|__|__|__|__|
2 | k|__|__|__|__|
1 |__|__|__|+R|__|
    a  b  c  d  e

Captures UPPER: G S B P
Captures lower: p b s g

lower> 
